runners/wikiConsole/index.ts

```diff
@@ -29,4 +30,15 @@ export class WikiConsole extends WikiRunner {
         this.renderWiki(path.join(this.getRunnerDirectory(), "templates", "cloneRepository.asciidoc"), { directoryPath: directoryPath, url: runCommand.command.parameters[1] });
         return null;
     }
+
+    runDownloadFile(runCommand: RunCommand): RunResult{
+        let url = runCommand.command.parameters[0];
+        let fileName = runCommand.command.parameters[1];
+        let dir = runCommand.command.parameters[2];
+        let directoryExists = dir 
+        ? fs.existsSync(path.join(this.getWorkingDirectory(), dir))
+        : undefined;
```

Comment:
The wiki runner does not really create files or folders on the file system. So this is not needed I think

Reply:
I want to check if the directory exists that I gonna download the file into.

Reply:
But the wiki runner only generate some description that the user can follow and execute later. So there is no need to check if the directory exists when the tutorial is executed in the compiler

Reply:
I have removed it.